### PR TITLE
Add SwaggerUtils for Swagger 2.0 (restlet/apispark#3793)

### PR DESCRIPTION
**Pull request by quilicicf**

### CODE REVIEW

File: modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/ImportUtils.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+     * @return The de-serialized content.
+     * @throws TranslationException
+     */
+    protected static <T> T getAndDeserialize(String url, String userName, String password,
+            Class<T> clazz) throws TranslationException {
+        try {
+            ClientResource cr = createAuthenticatedClientResource(url, userName, password);
+            Representation representation = cr.get();
+
+            representation.setMediaType(MediaType.APPLICATION_JSON);
```
Comment (boillodmanuel) [inline modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/ImportUtils.java]: //need comment

**Comment by quilicicf** [comment]
Reviewed with MBO

File: libraries/com.wordnik.swagger.core_1.5/.classpath [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
-	<classpathentry exported="true" kind="lib" path="com.wordnik.swagger.core.jar"/>
+	<classpathentry exported="true" kind="lib" path="com.wordnik.swagger.core.jar" sourcepath="/home/cyp/.m2/repository/com/wordnik/swagger-core/1.5.3-M1/swagger-core-1.5.3-M1-sources.jar"/>
```
Comment (boillodmanuel) [inline libraries/com.wordnik.swagger.core_1.5/.classpath]: absolute path!

File: modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/ImportUtils.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+     * @param clazz
+     *            The Java class in which to de-serialize.
+     * @return The de-serialized content.
+     * @throws TranslationException
+     */
+    public static <T> T getAndDeserialize(String url, String userName, String password,
+            Class<T> clazz) throws TranslationException {
+
+        Representation representation;
+        try {
```
Comment (boillodmanuel) [inline modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/ImportUtils.java]: try catch useless -> remove it

**Pull request merged**